src/main.ts

```diff
@@ -65,6 +66,27 @@ export function portMap(channels: MessageChannels, portNumber: 1 | 2): MessagePo
 }
 
 export function makeSandboxDriver(): Driver<undefined, Sources> {
+
+
+  let raf: boolean;
+  
+  const raf$ = xs.create({
+    start(listener) {
+      raf = true;
+      const r = () => requestAnimationFrame((e) => {
```

Comment:
Well the problem is that the DOMConnector is only going to work in a browser anyway.
The DOMConnector is initialised in both the worker and the main, it should know of this injection both places.

I don't want to move the raf$ signal into the DOMConnector because the raf$ signal is the same regardless if there are many instances of the same component, that is why I have it one subscription per worker, not per component.

I guess a solution could be to make it an option as you suggest, and then send the option to the worker thread, and somehow tell the DOMConnector down there, that it should expect a raf$ signal.